Formula/tmux.rb

```diff
@@ -32,11 +34,16 @@ class Tmux < Formula
   def install
     system "sh", "autogen.sh" if build.head?
 
+    configure_args = [
+      "--disable-Dependency-tracking",
+      "--prefix=#{prefix}",
+      "--sysconfdir=#{etc}",
+    ]
```

Comment:
Well there's no point in enforcing everything. Experienced human beings are watching this repo, and should be able to point out things like this. (Also, I doubt you can ask rubocop to flag `configure_args` in favor of `args`.)